workspaces/liferay-dlfoldertemplate-workspace/client-extensions/liferay-dlfoldertemplate-custom-element/src/utils/appUtil.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+function suggestServerUrl() {
+	const OAuthApp = getLiferay().OAuth2Client.FromUserAgentApplication(
+		getConfig().agentOauthAppId
+	).homePageURL;
+
+	return OAuthApp;
+}
+function getConfig() {
+	const config = {
+		'agentOauthAppId':
+			'liferay-dlfoldertemplate-oauth-application-user-agent',
+		'apiHost': 'http://localhost:8080',
+		'external': false,
+		'folder.generate.service.url': 'jobs/create/folder/direct',
+		'templateInfoApi': 'o/c/t4t14foldertemplates',
+		'templateNodeApi': 'o/c/t4t14foldertemplatenodes',
+	};
+
+	return config;
+}
+async function getOAuthToken() {
+	const prom = new Promise((resolve, reject) => {
+		getLiferay()
+			.OAuth2Client.FromUserAgentApplication(getConfig().agentOauthAppId)
```

Comment:
yeh i saw that example, however, the sample is running a simple fetch method and i dont think its mandatory since its already has some bugs, in my micro service, i am publishing a custom end point which does not fall under any type of backend cx, so i would prefer to use axios to call it and pass the token manually without Liferay util, there is an open bug about the same util with post 

 https://liferay.atlassian.net/browse/LPS-202620?atlOrigin=eyJpIjoiOTE3OTIwZmVkNDNlNDYzYzhmM2E3M2ZlNmExZjM3NzgiLCJwIjoiamlyYS1zbGFjay1pbnQifQ